ignite/metrics/rouge.py

```diff
@@ -0,0 +1,151 @@
+from collections import defaultdict
+from typing import Callable, DefaultDict, Dict, List, Optional, Union
+
+import torch
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics import Metric
+
+# These decorators helps with distributed settings
+from ignite.metrics.metric import reinit__is_reduced, sync_all_reduce
+
+
+class Rouge(Metric):
+    r"""Calculates the Rouge Score for two Sequence of Tokens.
+
+
+    .. math::
+        F_\1 = \left( 1 - \alpha \right) * \frac{ \text{precision} * \text{recall} }
```

Comment:
```suggestion
        F_\alpha = \left( 1 - \alpha \right) * \frac{ \text{precision} * \text{recall} }
```

Reply:
f1 = (precision*recall)/((1-alpha)*precision + alpha*recall)
This is the actual formula. My mathematical notation could be wrong.

Reply:
See for instance https://en.wikipedia.org/wiki/F-score

`f1` can't depend on a parameter `alpha`. In original paper, the author uses `f_\beta` score.